docs/device-and-supply-chain-modelling.md

```diff
@@ -0,0 +1,76 @@
+
+### Hardware Group
+A device MAY contain a Hardware Group ID that is presented in attestation evidence. This is an identifier, provisioned during manufacturing, which can be used to classify the SoC used within the device and will be the same for any devices built using that SoC. Although termed as a hardware ID, in fact this identifier will classify the immutable Root of Trust, being the un-modifiable and un-measured initial BIOS as well as the specific SoC hardware within the SoC. The identifier is a public value and has no security value, but aids classification for a verifier. If no Hardware Group ID is included within the attestation evidence then an equivalent value can be provisioned into the verifier as a correlation to the Trust Anchor ID.
+
+### Firmware
+A device is ASSUMED to produce evidence allowing the set of loaded firmware components to be identified. In some cases the device may summarise all loaded components into a single version number. In other cases a full set of measurements and metadata may be be made available to identify the set of firmware components loaded.
+
+### Workload
+An attestation report may include a series of measurements for the purpose of describing the workload running on a device. While a Veraison based attestation service can be capable of verifying such a series, there may not be a standard model for the reference values needed for this work. The workload is normally also divorced from the TCB and may require verification data to be obtained from a different supply chain from that used for device verification. This classification is therefore excluded from the device modelling assumptions.
+
+### Other Evidence
+Other evidence may be contained within the Attestation Report that does not fall into the above categories. This evidence may need to be referenced as part of an evidence appraisal policy, but no additional classification for a group of evidence is known at the moment to affect the modelling of Endorsements for the project.
+
+Note: attestation reports may contain additional complexities to modelling evidence, such as having multiple attesters within a device or receiving an encrypted report. However, it is expected that these can be handled as precursors to report verification by deconstruction into core parts in a pre-verification pass.
+
+Diagram: relevant groupings for evidence
+
+![Groupings](http://www.plantuml.com/plantuml/proxy?src=https://raw.githubusercontent.com/veraison/veraison/main/docs/diags/tokens-model.puml)
+
+Diagram: Known Tokens
+
+![Diagram: Mapping known attestation tokens to the evidence classifications](http://www.plantuml.com/plantuml/proxy?src=https://raw.githubusercontent.com/veraison/veraison/main/docs/diags/known-models.puml)
+
+
+## Firmware Lifecycles
+Where an attestation report contains only a firmware version, no additional data is required from the supply chain and whether the value provided is acceptible is a matter for the evidence appraisal policy.
```

Comment:
Even though this is not the case for PSA, the SW version could also be a reference value that is checked against the presented Evidence.

Reply:
I had something along those lines and linked it to arbitrary queries for the 'other' section but then removed it as it sounded too vague and didn't influence the data model.

Reply:
What I'm saying is that we might want to avoid special-casing this and treat version (or other more or less complex SW identifiers) as reference values in the same class as measurements.

Reply:
OK, I'll ponder some text restoring some reference to RVs. I really wanted just to talk about the modelling required for measurements etc here & only included a reference to version roll up to respect that subset.